docs/technical_reference/property_models/activated_sludge_model.rst

```diff
@@ -0,0 +1,144 @@
+ASM1 Property Package
+=====================
+
+This package implements property relationships for the treatment of activated sludge as provided in `Alex, J. et al. (2008) <https://www.iea.lth.se/publications/Reports/LTH-IEA-7229.pdf>`_.
+
+This Activated Sludege Model no.1 (ASM1) property package:
+   * supports only 'H2O', 'TDS', 'TSS', and 'Sludge' as Components
+   * supports only liquid phase
+   * is formulated on a mass basis
+   * does NOT support formulations on a molar basis
+   * includes mass density correction for fraction of suspended/dissolved solids
+
+Sets
+----
+.. csv-table::
+  :header: "Description", "Symbol", "Indices"
+
+  "Components", ":math:`j`", "['H2O', 'TDS', 'TSS', 'Sludge']"
+  "Phases", ":math:`p`", "['Liq']"
+
+State variables
+---------------
+.. csv-table::
+   :header: "Description", "Symbol", "Variable", "Index", "Units"
+
+   "Component mass flowrate", ":math:`M_j`", "flow_mass_phase_comp", "[p, j]", ":math:`\text{kg/s}`"
+   "Temperature", ":math:`T`", "temperature", "None", ":math:`\text{K}`"
+   "Pressure", ":math:`P`", "pressure", "None", ":math:`\text{Pa}`"
```

Comment:
You'll want to review the state variables as defined in the ASM1 property model in PR #502 and update these.